examples/window-app/common/src/WindowApp.cpp

```diff
@@ -245,14 +245,19 @@ void WindowApp::DispatchEvent(const WindowApp::Event & event)
             EmberAfStatus status;
             NPercent100ths current;
             chip::EndpointId endId = GetCover().mEndpoint;
-            chip::Percent100ths targetLift;
+            chip::Percent100ths target;
             chip::DeviceLayer::PlatformMgr().LockChipStack();
             status = Attributes::CurrentPositionLiftPercent100ths::Get(endId, current); 
```

Comment:
this seems better yes but you aren't reading the right attribute in the case of a tilt since you only read LiftPercent100ths then do the if tilt mode.

Same with the other button event.

would probably better to wrap this in a function like it was before and use the same function for both buttons event instead of duplicating code